src/test/java/org/opentest4j/AssertionFailedErrorTests.java

```diff
@@ -113,15 +133,33 @@ public void serializationWorksForAssertionFailedErrorWithMessageAndExpectedAndAc
+
+		AssertionFailedError error = deserializeAndCheck(Base64.decodeBase64(serializedVersion_1_0_0));
```

Comment:
I agree with @sormuras and @kcooney: loading the serialized object from a classpath resource would be more _conventional_ and preferable here. 😉